apps/roam/src/components/settings/LeftSidebarPersonalSettings.tsx

```diff
@@ -0,0 +1,551 @@
+import discourseConfigRef from "~/utils/discourseConfigRef";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import FlagPanel from "roamjs-components/components/ConfigPanels/FlagPanel";
+import AutocompleteInput from "roamjs-components/components/AutocompleteInput";
+import getAllPageNames from "roamjs-components/queries/getAllPageNames";
+import { Button, Dialog, Collapse, Tooltip } from "@blueprintjs/core";
+import createBlock from "roamjs-components/writes/createBlock";
+import deleteBlock from "roamjs-components/writes/deleteBlock";
+import type { RoamBasicNode } from "roamjs-components/types";
+import NumberPanel from "roamjs-components/components/ConfigPanels/NumberPanel";
+import TextPanel from "roamjs-components/components/ConfigPanels/TextPanel";
+import { LeftSidebarPersonalSectionConfig } from "~/utils/getLeftSidebarSettings";
+import { extractRef, getSubTree } from "roamjs-components/util";
+import getTextByBlockUid from "roamjs-components/queries/getTextByBlockUid";
+import getPageUidByPageTitle from "roamjs-components/queries/getPageUidByPageTitle";
+import { DISCOURSE_CONFIG_PAGE_TITLE } from "~/utils/renderNodeConfigPage";
+import getCurrentUserDisplayName from "roamjs-components/queries/getCurrentUserDisplayName";
+import { getLeftSidebarPersonalSectionConfig } from "~/utils/getLeftSidebarSettings";
+
+const SectionItem = React.memo(
+  ({
+    section,
```

Comment:
There's a lot of `props` here.

I wonder if we could think about what specifically needs to be here vs what could be fully moved down into `SectionItem`